src/main/java/space/obminyashka/items_exchange/security/jwt/refresh/RefreshTokenRepository.java

```diff
@@ -0,0 +1,14 @@
+package space.obminyashka.items_exchange.security.jwt.refresh;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.Optional;
+
+public interface RefreshTokenRepository extends JpaRepository<RefreshToken, Long> {
```

Comment:
Prefer having that into repository layer.

Reply:
Ok